instrumentation/aws-lambda/aws-lambda-core-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambdacore/v1_0/internal/AwsLambdaFunctionInstrumenter.java

```diff
@@ -22,12 +22,16 @@
 public class AwsLambdaFunctionInstrumenter {
 
   private final OpenTelemetry openTelemetry;
+  private final CarrierEnricher carrierEnricher;
```

Comment:
it looks like constructor param not needed?

```suggestion
  private final CarrierEnricher carrierEnricher = new AwsXrayEnvCarrierEnricher();
```